test/Microsoft.NET.Build.Tests/GivenThatWeWantToTargetNotRecommendedFrameworks.cs

```diff
@@ -0,0 +1,93 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.NET.Build.Tests
+{
+    public class GivenThatWeWantToTargetNotRecommendedFrameworks : SdkTest
+    {
+        public GivenThatWeWantToTargetNotRecommendedFrameworks(ITestOutputHelper log) : base(log)
+        {
+        }
+
+        [Theory]
+        [InlineData("NetStandard1.0")]
+        [InlineData("NetStandard1.1")]
+        [InlineData("NetStandard1.2")]
+        [InlineData("NetStandard1.3")]
+        [InlineData("NetStandard1.4")]
+        [InlineData("NetStandard1.5")]
+        [InlineData("NetStandard1.6")]
+        public void It_warns_that_framework_is_not_recommended(string targetFrameworks)
```

Comment:
Do we have any coverage that the warning is not generated for .NET Standard 2.0 or 2.1?